front/src/components/common/ArchivePopup.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled';
+import { Flex, Text } from '.';
+import popup from '@assets/images/popup.svg';
+import { useEffect, useState } from 'react';
+
+export const ArchivePopup = ({ option }: { option: string }) => {
+  const [popupClose, setPopupClose] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setPopupClose(false);
+    }, 3000);
+  }, []);
```

Comment:
```suggestion
let popupTimer = null;
  useEffect(() => {
    clearTimeout(popupTimer)
    popupTimer = setTimeout(() => {
      setPopupClose(false);
    }, 3000);

  }, []);
```

와 같이 초기화 로직을 넣어주면 좋을 것 같아요!! 타이머 쓸 때는 항상 중첩 이슈가 생기더라구요 ㅜ

Reply:
오 감사합니다. 반영완료했습니다!!